Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>double word? <code>subset subset</code>  this is a confusing line in general. what are we dont next?</p><p>dont say "those series" say which series and what they are for </p><p>"region we need"? what region? for what?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Fixed</p>